app/css/app.css

```diff
@@ -97,43 +95,42 @@ html, body {
   margin: 20px 5px;
   margin-right: 75px;
 }
-.menu .shao-sing-up:hover {
+.main .menu__shao-sing-up:hover {
```

Comment:
must be just .nemu__shao-sing-up

https://ru.bem.info/methodology/naming-convention/